pkgs/by-name/ma/macos-instantview/package.nix

```
@@ -0,0 +1,44 @@
+{
+  stdenvNoCC,
+  fetchurl,
+  lib,
+  _7zz,
+}:
+
+stdenvNoCC.mkDerivation (self: {
```

nit: best to follow convention here
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```